clcache.py

```diff
@@ -7,6 +7,7 @@
 # root directory of this project.
 #
 from ctypes import windll, wintypes
+import cProfile
```

Comment:
The disk cache may have something to do with it, but it's also the fact that the Python interpreter stores the byte code of the module to disk and reuses that. Try creating e.g. a file `a.py` with

```python
import b
```

and `b.py` with e.g.

```python
def f(): pass
```

and then run `python a.py`. Notice how `b.pyc` is created - it's a binary version of `b.py`.